PowerShell Scanners/Windows Firewall Rules/Windows Firewall Rules.ps1

```diff
@@ -0,0 +1,19 @@
+[cmdletBinding()]
+param(
+    [parameter()]
+    [switch]
+    $Enabled
+)
+
+process{
+    
+    if([version](Get-WmiObject win32_operatingsystem).Version -lt 6.3.0){
+        throw "This scanner only available on Server 2012/Windows8 or higher"
+    }
+    
+    switch($Enabled){
+        $true { Get-NetFirewallRule -Enabled True| Select-Object Name,DisplayName,Description,Profile,Direction,Action }
```

Comment:
I'd like to see a space after `True` to match the formatting of the line below.

Reply:
Fixed!